geojson_pydantic/features.py

```diff
@@ -22,9 +21,9 @@ class Feature(GenericModel, Generic[Geom, Props], GeoInterfaceMixin):
     id: Optional[Union[StrictInt, StrictStr]] = None
     bbox: Optional[BBox] = None
 
-    _validate_bbox = validator("bbox", allow_reuse=True)(validate_bbox)
+    _validate_bbox = field_validator("bbox")(validate_bbox)
 
-    @validator("geometry", pre=True, always=True)
+    @field_validator("geometry", mode="before")
```

Comment:
Was dropping `always=True` here intentional? I think that may match the other validators better, but wanted to verify the thought process.

Reply:
no 🤦

Reply:
Looked like this was the only one which had `always=True` on it. I just realized `field_validator` doesn't have the kwarg anymore. It still exists on the deprecated `@validator` though. Docs aren't completely clear as to what is going on there.  I don't think it makes a difference since its a required field, so there isn't a default to validated. I guess it could make a difference if someone inherits from the class and sets a bad default? Don't know its worth dealing with that though.